pkg/client/leaderelection/leaderelection.go

```diff
@@ -21,9 +21,10 @@ limitations under the License.
 // This implementation does not guarantee that only one client is acting as a
 // leader (a.k.a. fencing). A client observes timestamps captured locally to
 // infer the state of the leader election. Thus the implementation is tolerant
-// to arbitrary clock skew, but is not tolerant to arbitrary clock skew rate.
+// to arbitrary clock offset, but is not tolerant to arbitrary clock skew (the
```

Comment:
Not sure who to trust

>  A clock’s skew at time T is its frequency F at time T minus true frequency

from https://www.iol.unh.edu/sites/default/files/knowledgebase/1588/clock_synchronization_terminology.pdf

I'll take a broader survey today